omnibus/config/software/opscode-solr4.rb

```diff
@@ -54,4 +52,10 @@
   delete "#{service_dir}/jetty/example*"
   delete "#{service_dir}/jetty/multicore"
   delete "#{service_dir}/jetty/solr"
+
+  # Replace the built-in admin.html with a blank page to effectively disable the admin
+  # interface to solr.  Note that this does not disable API access.
+  touch "admin.html"
+  command "jar -uf #{service_dir}/jetty/webapps/solr.war admin.html"
```

Comment:
Reviewer note - looks like I'll put that `env` back in and use it here so that `jar` can be found at build time...